gradle/libs.versions.toml

```diff
@@ -0,0 +1,71 @@
+[versions]
+androidGradlePlugin = "7.4.1"
+androidxActivity = "1.6.1"
+androidxComposeBom = "2023.01.00"
+androidxComposeCompiler = "1.4.0"
+androidxComposeMaterial3 = "1.0.1"
+androidxComposeMaterialIconsExtended = "1.3.1"
+androidxComposeUi = "1.3.3"
+androidxCore = "1.9.0"
+androidxLifecycle = "2.6.0-alpha03"
+androidxLifecycleProcess = "2.5.1"
+coil = "2.2.2"
+coroutines = "1.6.4"
+espresso = "3.5.1"
+junit = "1.1.5"
+junit4 = "4.13.2"
+kermit = "1.2.2"
+kotlin = "1.8.0"
+kotlinxCoroutines = "1.6.4"
+#ksp = "1.8.0-1.0.8"
+ktor = "2.2.2"
+loginoauth = "0.11.1"
+multiplatform-settings = "1.0.0"
+timber = "5.0.1"
+
+[libraries]
+android-gradlePlugin = { group = "com.android.tools.build", name = "gradle", version.ref = "androidGradlePlugin" }
+androidx-activity-compose = { group = "androidx.activity", name = "activity-compose", version.ref = "androidxActivity" }
+
+# uses compose-bom so no version.ref for some deps
+androidx-compose-bom = { group = "androidx.compose", name = "compose-bom", version.ref = "androidxComposeBom" }
```

Comment:
If we're using the BOM should we remove the `version.ref` for most of the compose dependencies below? @oneSIX

Reply:
Done.